static/js/theme-map/CardListenerAssigner.js

```diff
@@ -9,6 +9,20 @@ class CardListenerAssigner {
      * @type {Answers.Component}
      */
     this.card = card;
+
+    /**
+     * The upper bound (inclusive) of the mobile breakpoint in px
+     *
+     * @type {Number}
+     */
+    this.mobileBreakpointMax = 991;
```

Comment:
would we ever use this variable directly as opposed to this.mediaMatcher?

Reply:
We would not! Removed the one-use variable